foundation/erpnext_foundation/doctype/service_provider/templates/service_provider.html

```diff
@@ -1,25 +1,22 @@
 {% extends "templates/web.html" %}
 
 {% from "foundation/templates/includes/macros.html" import media %}
-
 {% block page_content %}
 <h1 class="media-heading">{{ title }}</h1>
 {% if membership_type %}
 <p><span class=label>{{ membership_type }} Member<span></p>
 {% endif %}
 <section>
-	<div class="media" style="margin: 45px 0px; ">
-		<div class="media-left" style='min-width: 80px;'>
+	<div class="media">
+		<div class="media-left">
 			<a href="#">
 				<img class="media-object"
-					src="{{ image }}"
-					style="max-width: 80px; ">
+					src="{{ image }}">
```

Comment:
you can use this which use for fellowship.  src="{% if image %}{% if not '/' in image %}/assets/foundation/img/{% endif %}{{ image }}{% else %}/assets/foundation/img/frappe-bird-grey.svg{% endif %}"

Reply:
This is covered in the service provider.py itself, if you see [here](https://github.com/raghukamath/foundation/blob/40ec04f63497d09b3aeebf027b94f277a6f9b983/foundation/erpnext_foundation/doctype/service_provider/service_provider.py#L26-L32) you'll see that if the user is gold member, silver or individual member and not added image, the image variable gets default image links.